src/browser/browser.js

```diff
@@ -182,8 +189,14 @@ define((require, exports, module) => {
                }
               }),
 
+      LargeTile.Deck({key: 'dashboard',
+                      className: 'dashboard',
+                      hidden: !dashboardIsActive,
+                      items: dashboardItemsCursor}),
+
```

Comment:
It could be that not even creating a `LargeTile.Deck` would be a better option, I guess depends how often we're going to be changing `dashboardItems`.